src/ViewArticle/ViewArticleStyles/ViewArticle.scss

```diff
@@ -0,0 +1,95 @@
+@import '../../commons/assets/variables/font.scss';
+@import '../../commons/assets/variables/colors.scss';
+
+.article_title{
+    color: #00a4d3;
+    font-family:$source-san-font;
+    font-weight: light;
+    font-size: 40px;
+    margin-top:5px;
+}
+.myimage{
+    width: 90%;
+    margin-left: 5%;
+    margin-right: 5%;
+    height:259px;
```

Comment:
Here instead of fixing the `height` & `width` to a single value. I suggest instead specify the `max-height` an image can be to prevent small images from overstretching like they're right now. you may use `max-height:400px` & `max-width:100%`.

Reply:
okay let me work on this as well

Reply:
kindly noted l have worked on this thank you

Reply:
this is done thanks you